src/settings/Settings.tsx

```diff
@@ -167,8 +168,31 @@ export const SETTINGS: {[setting: string]: ISetting} = {
             feedbackSubheading: _td("Your feedback will help make spaces better. " +
                 "The more detail you can go into, the better."),
             feedbackLabel: "spaces-feedback",
+            extraSettings: [
+                "feature_spaces.all_rooms",
+                "feature_spaces.space_member_dms",
+                "feature_spaces.space_dm_badges",
+            ],
         },
     },
+    "feature_spaces.all_rooms": {
+        displayName: _td("Use an all rooms space instead of a home space."),
+        supportedLevels: LEVELS_FEATURE,
+        default: true,
+        controller: new ReloadOnChangeController(),
+    },
+    "feature_spaces.space_member_dms": {
+        displayName: _td("Show DMs for joined/invited members in the space."),
```

Comment:
```suggestion
        displayName: _td("Show DMs for joined/invited members in spaces."),
```

Maybe I am missing something here, but the original sounded a bit weird to me